deploy/operator/ztp/add_day2_local_cluster_nodes.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -x
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${__dir}/../common.sh"
+source "${__dir}/../utils.sh"
+
+export REMOTE_BAREMETALHOSTS_FILE="${REMOTE_BAREMETALHOSTS_FILE:-/home/test/dev-scripts/ocp/ostest/remote_baremetalhosts.json}"
+export LOCAL_CLUSTER_NAMESPACE="${LOCAL_CLUSTER_NAMESPACE:-local-agent-cluster}"
+
+echo "Adding day2 local cluster nodes"
+ansible-playbook "${__dir}/add-worker-nodes-to-local-cluster-playbook.yaml"
+
+comma_sep_host_names=$(jq -r '[.[].name] | join(",")' "${REMOTE_BAREMETALHOSTS_FILE}")
+export comma_sep_host_names
+
+if [ -z "${comma_sep_host_names}" ] ; then
+  echo "Missing bmhs names"
+  exit 1
+fi
+
+function remote_agents() {
+	oc get agent -n ${LOCAL_CLUSTER_NAMESPACE} --no-headers -l "agent-install.openshift.io/bmh in ( ${comma_sep_host_names} )"
+}
```

Comment:
we're just keep repeating this piece of code on some other places (with only the namespace being different)
can't we just move it to utils and re-use it?

Reply:
The only problem with this refactor for me is that I need to use the local cluster namespace and not the spoke namespace.
To move this code to a common place will require me to parameterize the namespace, in bash this is a bit weird and you have to use the $1 syntax inside the function.
I will need to test that, which will add complexity and time to my PR, I actually can't really test it unless I run it, so it would take hours
I'd rather leave this redundancy in for now, I see your point but it will be a fair amount of hassle because of the testing involved, existing tests would need to be modified to change the way in which the namespace is passed (edited)

Reply:
OK, changed